spec/classes/puppet_spec.rb

```diff
@@ -127,13 +127,11 @@
           should contain_file("#{facterbasepath}").with({
             :ensure=>"directory",
             :owner=>"root",
-            :group=>"puppet",
```

Comment:
Why are you removing the test for this parameter?
The test should be adjusted to test for the two different values this can have rather than not testing for it at all.